platform_versioned_docs/version-24.1.0/compute-envs/azure-batch.mdx

```diff
@@ -205,13 +205,25 @@ Your Seqera compute environment uses resources that you may be charged for in yo
+13. Define custom **Environment Variables** for the **Head Job** and/or **Compute Jobs**.
+14. Configure any necessary advanced options:
 
     - Use **Jobs cleanup policy** to control how Nextflow process jobs are deleted on completion. Active jobs consume the quota of the Azure Batch account. By default, jobs are terminated by Nextflow and removed from the quota when all tasks succesfully complete. If set to _Always_, all jobs are deleted by Nextflow after pipeline completion. If set to _Never_, jobs are never deleted. If set to _On success_, successful tasks are removed but failed tasks will be left for debugging purposes.
     - Use **Token duration** to control the duration of the SAS token generated by Nextflow. This must be as long as the longest period of time the pipeline will run.
 
-13. Select **Add** to finalize the compute environment setup. It will take a few seconds for all the resources to be created before you are ready to launch pipelines.
+15. Click **Add** to complete the compute environment setup. The creation of resources will take a few seconds, after which you can launch pipelines.
+
+### Managed Identity
+
+Nextflow can authenticate to Azure services using a Managed Identity, offering enhanced security compared to access keys, but this method requires running on Azure infrastructure. When using the Seqera Platform with a manually configured compute environment that has a Managed Identity attached to the Azure Batch Pool, Nextflow can utilize this Managed Identity for authentication. However, the Seqera Platform still needs to use access keys to submit the initial task to Azure Batch to run Nextflow, which will then proceed with the Managed Identity for subsequent authentication.
+
+1. In Azure, create a user-assigned Managed Identity. Refer to the [Azure documentation](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/how-manage-user-assigned-managed-identities) for detailed steps. After creation, make sure to record the Client ID of the Managed Identity.
+2. Ensure the user-assigned Managed Identity has the necessary roles for Nextflow. Consult the [Nextflow documentation](https://www.nextflow.io/docs/latest/azure.html#required-role-assignments) for role requirements.
+3. Associate the user-assigned Managed Identity with the Azure Batch Pool. Detailed instructions can be found in the [Azure documentation](https://learn.microsoft.com/en-us/troubleshoot/azure/hpc/batch/use-managed-identities-azure-batch-account-pool#set-up-managed-identity-in-your-batch-pool).
+4. When setting up the Seqera Platform Compute Environment, select the Azure Batch Pool by name and enter the Managed Identity Client ID in the specified input field as instructed above.
+
+After submitting a pipeline to this compute environment, Nextflow will authenticate using the Managed Identity associated with the Azure Batch node it is running on, rather than relying on access keys.
```

Comment:
```suggestion
When you submit a pipeline to this compute environment, Nextflow will authenticate using the Managed Identity associated with the Azure Batch node it runs on, rather than relying on access keys.
```